package.json

```diff
@@ -50,6 +50,7 @@
     "browserify": "^15.2.0",
     "chai": "^4.1.2",
     "coveralls": "^3.0.0",
+    "cypress": "^1.4.1",
```

Comment:
where is cypress downloading its binary to on travis? I’d add that folder to caching